src/sentry/utils/pytest/fixtures.py

```diff
@@ -0,0 +1,224 @@
+                    'k': 'logentry',
+                    'public_key': None,
+                    'result': {
+                        'logentry':
+                        "{'message': 'This is a test message generated using ``raven test``', 'params': []}"
+                    },
+                    'self': '<raven.base.Client object at 0x104397f10>',
+                    'stack': True,
+                    'tags': None,
+                    'time_spent': None,
+                },
+            },
+            {
+                'abs_path':
+                '/Users/dcramer/.virtualenvs/sentry/lib/python2.7/site-packages/raven/base.py',
+                'context_line':
+                '                        string_max_length=self.string_max_length)',
+                'filename':
+                'raven/base.py',
+                'function':
+                'build_msg',
+                'in_app':
+                False,
+                'lineno':
+                290,
+                'module':
+                'raven.base',
+                'post_context': [
+                    '                },', '            })', '',
+                    "        if 'stacktrace' in data:",
+                    '            if self.include_paths:'
+                ],
+                'pre_context': [
+                    '', '            data.update({',
+                    "                'stacktrace': {",
+                    "                    'frames': get_stack_info(frames,",
+                    '                        list_max_length=self.list_max_length,'
+                ],
+                'vars': {
+                    'culprit': 'raven.scripts.runner',
+                    'date': 'datetime.datetime(2013, 2, 14, 20, 6, 33, 479471)',
+                    'event_id': '598fb19363e745ec8be665e6ba88b1b2',
+                    'event_type': 'raven.events.Message',
+                    'frames': '<generator object iter_stack_frames at 0x103fef050>',
+                    'handler': '<raven.events.Message object at 0x103feb710>',
+                    'k': 'logentry',
+                    'public_key': None,
+                    'result': {
+                        'logentry':
+                        "{'message': 'This is a test message generated using ``raven test``', 'params': []}"
+                    },
+                    'self': '<raven.base.Client object at 0x104397f10>',
+                    'stack': True,
+                    'tags': None,
+                    'time_spent': None,
+                },
+            },
+        ],
+    },
+    'tags': [],
+    'platform': 'python',
+}
+
+
+@pytest.fixture
+def factories():
+    # XXX(dcramer): hack to prevent recursive imports
+    from sentry.testutils.factories import Factories
+
+    return Factories
+
+
+@pytest.fixture(scope='function')
+def session():
+    return factories.create_session()
+
+
+@pytest.fixture(scope='function')
```

Comment:
these are all copypasta but it also decouples old style from new style meaning we could create or augment better defaults for newer tests